### DISCUSSION TITLE
Ошибка при включении, не может найти rundll32.

### DISCUSSION THREAD

**Discussion by CodelessHeart** [lirix360/ReadmangaGrabber#16] [General]
```
2021-09-02 20:50:26	[INFO]:	Запуск приложения!
2021-09-02 20:50:26	[FATAL]:	Ошибка при открытии браузера: exec: "rundll32": executable file not found in %PATH%

```

**Comment by CodelessHeart** [comment]
![image](https://user-images.githubusercontent.com/82465137/131964044-34aa26e2-d8e9-4823-b5ed-c93a53fa6d2c.png)
Сам rundll32 есть на пк, и функционирует.
Я думала что это изза брандмауэра, который блокирует открытие веб сервера, но после отключения , всё так же.

**Comment by lirix360** [comment]
Какая версия Windows используется?

**Comment by lirix360** [comment]
Проверьте beta 3 версию, будет ли в ней ошибка.
